OurUmbraco.Site/Views/MacroPartials/Projects/ProjectDetails.cshtml

```diff
@@ -51,15 +51,48 @@
     <div class="package-detail-content">
 
         <div class="vote">
-            <i class="icon-Hearts @(hasMemberVoted ? "liked" : null)"></i>
-            @if (Members.IsLoggedIn() == false || currentMember.Id == owner.Id || hasMemberVoted)
-            {
-                <span>@totalVotes votes</span>
-            }
-            else
-            {
-                <a href="#" id="projectVote" data-id="@project.Id">@totalVotes votes</a>
-            }
+            <i class="icon-Hearts @(hasMemberVoted ? "liked" : null)"></i>
+            
+            @if (Members.IsLoggedIn() == false || currentMember.Id == owner.Id)
+            {
+
+                if (totalVotes == 0 || totalVotes > 1)
+                {
+                    <span>@totalVotes votes</span>
+                }
+                else
+                {
+                    <span>@totalVotes vote</span>
+                }
+
+
+            }
+
+           @if (Members.IsLoggedIn() && hasMemberVoted)
+           {
+               if (totalVotes == 0 || totalVotes > 1)
+               {
+                <span>@totalVotes votes</span>
+               }
+               else
+               {
+                <span>@totalVotes vote</span>
+               }
+           }
+
+            @if (Members.IsLoggedIn() && !hasMemberVoted)
+            {
+                if (totalVotes == 0 || totalVotes > 1)

```

Comment:
You're repeating the exact same code 3 times here. Better to just make one variable like:
```
var voteText = "votes";
if(totalVotes == 1) { voteText = "vote"; } 
```
And then use `@voteText` everywhere.